lua/nvim-tree.lua

```diff
@@ -663,6 +663,13 @@ local function validate_options(conf)
   end
 end
 
+--- Apply OS specific localisations to DEFAULT_OPTS
+local function localise_default_opts()
+  if utils.is_macos then
+    DEFAULT_OPTS.trash.cmd = "trash"
```

Comment:
Shouldn't that be `gio trash`?

Reply:
This is where I get confused. I was under the impression that gio would put the files in `~/.local/share/Trash/files` which is not recognised by macos.

Perhaps we should just update the doc, removing "Trash is unavailable" and noting the available homebrew options for macos users.

Reply:
I think it might be time for me to use https://github.com/sickcodes/Docker-OSX

Reply:
[Here](https://github.com/nvim-tree/nvim-tree.lua/issues/1366#issuecomment-1646788804), we found that neither `trash` nor `gio` comes preinstalled on macOS so it's best to do what you said, which is to warn the users that `D` is not available unless you install `trash` from homebrew. We shouldn't use `gio trash` on macOS as it moves the file to a different location than `~/.Trash` which is where the Bin icon is pointing to.